scripts/verify_crop_recommendations.py

```
@@ -0,0 +1,356 @@
+            'timestamp': '2025-08-06',
+            'total_crops': len(results),
+            'passed': passed,
+            'warnings': warnings,
+            'critical_errors': critical_errors,
+            'results': results
+        }
+        
+        with open('test_reports/crop_recommendations_verification.json', 'w', encoding='utf-8') as f:
+            json.dump(report, f, indent=2, ensure_ascii=False)
+        
+        print(f"\n📄 Отчет сохранен: test_reports/crop_recommendations_verification.json")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)